.pre-commit-config.yaml

```diff
@@ -58,14 +58,10 @@ repos:
     rev: v1.11.2
     hooks:
       - id: mypy
-        name: mypy
-        entry: mypy
         language: python
-        types: [python]
-        args: []
+        pass_filenames: false
         require_serial: true
         additional_dependencies: ["types-typed-ast"]
-        exclude: tests/testdata| # exclude everything, we're not ready
```

Comment:
~~We should still exclude testdata right?~~ nvm, saw that everything happens in the pyproject.toml now.